tox.ini

```diff
@@ -63,8 +63,7 @@ deps =
     requests
     -r{toxinidir}/requirements.txt
 commands = 
-    ./update_bundle.py releases/latest/machine/kafka/bundle.yaml
-    ./update_bundle.py releases/latest/k8s/kafka-k8s/bundle.yaml
+    ./update_bundle.py releases/latest/kafka/bundle.yaml
 
 [testenv:integration-vm]
```

Comment:
nit: this could be changed to just `[testenv:integration]`, and updated on `ci.yaml` as well. But it's fine as it is.